packages/credentials/src/jwt.ts

```diff
@@ -189,18 +146,11 @@ export class Jwt {
 
     const signatureBytes = Convert.base64Url(encodedJwt.signature).toUint8Array();
 
-    const algorithmId = `${decodedJwt.header.alg}:${publicKeyJwk['crv'] || ''}`;
-    if (!(algorithmId in Jwt.algorithms)) {
-      throw new Error(`Verification failed: ${algorithmId} not supported`);
-    }
-
-    const { signer, options: signatureAlgorithm } = Jwt.algorithms[algorithmId];
-
-    const isSignatureValid = await signer.verify({
-      algorithm : signatureAlgorithm!,
+    const crypto = new CryptoApi();
```

Comment:
Should we just have is initialized once at the top?

Reply:
sounds good